contracts/src/instances/eeth/EETHTarget0.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { ERC20 } from "openzeppelin/token/ERC20/ERC20.sol";
+import { SafeERC20 } from "openzeppelin/token/ERC20/utils/SafeERC20.sol";
+import { HyperdriveTarget0 } from "../../external/HyperdriveTarget0.sol";
+import { IHyperdrive } from "../../interfaces/IHyperdrive.sol";
+import { EETH_HYPERDRIVE_KIND } from "../../libraries/Constants.sol";
+import { ILiquidityPool } from "etherfi/src/interfaces/ILiquidityPool.sol";
+import { EETHBase } from "./EETHBase.sol";
+
+/// @author DELV
+/// @title EETHTarget0
+/// @notice EETHHyperdrive's target0 logic contract. This contract contains
+///         all of the getters for Hyperdrive as well as some stateful
+///         functions.
+/// @custom:disclaimer The language used in this code is for coding convenience
+///                    only, and is not intended to, and does not, have any
+///                    particular legal or regulatory significance.
+contract EETHTarget0 is HyperdriveTarget0, EETHBase {
+    using SafeERC20 for ERC20;
+
+    /// @notice Initializes the target0 contract.
+    /// @param _config The configuration of the Hyperdrive pool.
+    /// @param _liquidityPool The Etherfi liquidity pool contract.
+    constructor(
+        IHyperdrive.PoolConfig memory _config,
+        ILiquidityPool _liquidityPool
+    ) HyperdriveTarget0(_config) EETHBase(_liquidityPool) {}
+
+    /// @notice Returns the instance's kind.
+    /// @return The instance's kind.
+    function kind() external pure override returns (string memory) {
+        _revert(abi.encode(EETH_HYPERDRIVE_KIND));
+    }
+
```

Comment:
I think we want to add the `vault` (or `liquidityPool`) getter to this file.

Reply:
When you add this getter, it would be good to add a test to `EETHHyperdriveTest.t.sol`. This is an example of the kind of test that I'm talking about: https://github.com/delvtech/hyperdrive/blob/2514eecb19e9b7b202b96aa849e3a779a9b3ab9b/test/instances/ezETH/EzETHHyperdrive.t.sol#L270.

Reply:
okay, i was just following the pattern from steth, ezeth, lseth, etc. None of them had a vault getter. Should I add it to the codegen?

Also, should I rename liquidityPool to vault?